cram_3d_world/cram_plan_transformation/cram-plan-transformation.asd

```diff
@@ -0,0 +1,81 @@
+;;;
```

Comment:
Get rid of the resource directory. Perhaps put the objects into cram_bullet_reasoning/resource. These are all copied from cram_pr2_pick_place_demo but some models in the demo are not copied here. Those that are not so often used. @artnie COPY PASTE IS A SIN! If there are two places where the same thing is used, make sure to put it in a general place where both can access, and not copy paste.